### ISSUE TITLE
Acquisition rate from FxA CTA on /firefox/new/ fell by 60% on Tuesday

### ISSUE THREAD

**Issue by hoosteeno** [mozilla/bedrock#6957]
## Description

We released a feature on /firefox/new/ that asks existing Fx users to sign up for an account. In our experiment, this was acquiring X users per day. When we graduated it, it was again acquiring X users per day. [On Tuesday, this dropped to appx. 0.3X users per day](https://analytics.amplitude.com/mozilla-corp/chart/dms0tq1). 

Did one of our changes on Tuesday change how many Firefox users see this treatment?

### Steps to reproduce

Visit the dashboard linked above.

### Expected result

Volume should remain fairly steady.

### Actual result

Volume dropped suddenly.

**Comment by hoosteeno** [comment]
This may just be a lag in data reporting. We can leave it for a few days to see if the numbers catch up.

**Comment by alexgibson** [comment]
Data is showing that we had a dip on Tuesday (83), but this went back up to more normal levels yesterday (208):

![image](https://user-images.githubusercontent.com/400117/54741282-eaaed180-4bb5-11e9-81b8-28a70b27eab9.png)

We have automated tests in place that confirm the CTA modal is shown when Firefox users click the button. These tests run prior to every deployment, so if something was amis I think we'd stand a good chance of being warned about it.

Something that's interesting about this timing is that this coincides with a new Firefox release. Looking in GA, we saw a bug spike around the same time of Firefox users re-downloading Firefox (i.e. not filling in the FxA form)

![image](https://user-images.githubusercontent.com/400117/54741667-0ff00f80-4bb7-11e9-8670-5888cc1354ad.png)

I read that we throttled the initial release to around 25% of people updating. My guess is people want the new release ASAP, so go to redownload rather than wait.
